pkg/server/plugin/datastore/sql/models.go

```diff
@@ -16,7 +17,7 @@ type Bundle struct {
 	Model
 
 	TrustDomain string `gorm:"not null;unique_index"`
-	Data        []byte
+	Data        []byte `gorm:"size:65535"`
```

Comment:
Is this a required constraint for MySQL? How did we arrive at 64K? The upper bound on this field is a function of how many SPIRE servers are running and the CA TTL.... I this there is potential to blow this limit.

Also, how will this impact other non-MySQL databases...?

Reply:
Without specifying the size, `data` column in MySQL was of type `VARBINARY(255)`. I picked the upper limit of `BLOB` and indeed it allows up to 64kB of data - it could have been any number > 255, and I figured that having this particular value would be good enough. We can set it to `MEDIUMBLOB` though - so anything between 65535 and 16777215 (24MB).

Regardless of this change, when I tried to initialize PostgreSQL (both on master and this branch), the `data` column was of type `BYTEA`. Similarly, Sqlite was setting `BLOB` in both cases, so no negative impact observed.

Reply:
I think I'd be a little more keen on a 24MB upper boundary. Unfortunately the failure mode in this scenario could be pretty terrible but it is what it is. We can revisit things at a later time if people start bumping up against this.